tests/test-sources/plugins/pluginmanagers/packer.nix

```diff
@@ -136,10 +137,19 @@
     };
   };
 
-  no-packages = {
+  no-packages = cfg: {
     plugins.packer = {
       enable = true;
       gitPackage = null;
     };
+
+    assertions = [
+      {
+        assertion =
+          cfg.config.extraPackages == [ ]
+          || lib.any (x: lib.trace "${x.pname or ""}" x.pname or null != "git") cfg.config.extraPackages;
```

Comment:
looks like we need to check using or when a derivation might not contain any extraPackages with the disabling of a package option

Reply:
Is this supposed to be using `lib.all`?

I.e. _all packages are not named "git"_

If so we shouldn't need to check for an empty list.

Reply:
Yeah, that'd be cleaner.